tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "compilerOptions": {
+    "target": "es6",
+    "lib": [
+      "es2016"
+    ],
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "isolatedModules": false,
+    "declaration": true,
+    "strict": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "suppressImplicitAnyIndexErrors": false,
+    "removeComments": false,
+    "preserveConstEnums": true,
+    "outDir": "dist",
```

Comment:
Will do after this set of initial conversion PRs have landed, want to keep the changes approximately minimal.